tez-api/src/test/java/org/apache/tez/client/TestTezClient.java

```diff
@@ -203,12 +203,12 @@ TezClientForTest configureAndCreateTezClient(Map<String, LocalResource> lrs, boo
   
   @Test (timeout = 5000)
   public void testTezclientApp() throws Exception {
-    testTezClient(false, true);
+    testTezClient(false, true, "testTezclientApp");
```

Comment:
I know this patch is not about that, but...: can you please fix all occurrences of "Tezclient" -> "TezClient"?
feel free to create a new ticket if you want